.github/workflows/windows.yml

```diff
@@ -76,6 +76,17 @@ jobs:
           architecture: x64
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      #
+      #      - name: Setup ALGLIB
+      #        run: |
+      #          git clone https://github.com/S-Dafarra/alglib-cmake.git
+      #          cp alglib-cmake ${{github.workspace}}/libs/
+      #          cd ${{github.workspace}}/libs/alglib-cmake
+      #          (Get-Content CMakeLists.txt) -replace '/Ox', '' | Set-Content CMakeLists.txt
+      #          mkdir build && cd build
+      #          cmake .. -G "${{matrix.env}}" -A x64 -T ${{matrix.toolset}}
+      #          cmake --build . --target ALL_BUILD --config ${{matrix.build_type}} -j4
+      #          cmake --build . --target INSTALL --config ${{matrix.build_type}}
```

Comment:
Is this old?

Reply:
If you look at the commit messages earlier you will find a light form of psychosis embedded, originating from a variety of issues I was facing getting all the Windows CI builds to work with ALGLIB. I gave up in the end since it was not worth my time and my sanity. Hopefully I can integrate CC without the ALGLIB dependency later on by extracting the code from ALGLIB, but my god this code base is so intensely interconnected, written in a C-style fashion with 0 whitespace between definitions. Intended on doing it but I decided not to, saving my sanity for now (again).